doc/podboat-cmds.dsv

```diff
@@ -1,3 +1,4 @@
 download-path||<path>||~/||Specifies the directory where podboat shall download the files to. Optionally, the placeholders "%n" (for the podcast feed's name) and "%h" (for the podcast feed's hostname) can be used to place downloads in a directory structure.||download-path "~/Downloads/%h/%n"
+download-filename||<string>||"%?u?%u&%Y-%b-%d-%H%M%S.unknown?"||Specifies the format of the downloaded file. Use the placeholders "%t" (for the podcast episode's title) and %e (for the filename's extension). For date/time formats, consult the manpage of strftime(3). Supported options are "%F", "%m", %b", "%d", "%H", "%M", "%S", "%y" and "%Y".||download-filename "%F-%t.%e"
```

Comment:
> Specifies the format of the downloaded file.

A user reading this will probably think that this chooses between MP3, OGG and whatnot—it's the format of the file, isn't it? I'd write "Specifies how Podboat would name the files it downloads".

I'd also add "See also `download-path`" at the end. If we link to the setting from the web, user might not look around the table to see what related settings there are.

Finally, I apologize for asking for this so late, but let's rename the setting to "download-filename-format". This is more in line with how we name format settings. (Pity it's too late to fix that for "download-path").

Reply:
I see that the common pattern is to mention "(see `download-path`)" at the end. Is it fine if I do it that way or do you want to change that?

Reply:
I think it's fine. Consistency is always nice :)